lib/games.rb

```diff
@@ -0,0 +1,25 @@
+class Games
```

Comment:
This set up looks great. Makes it very readable.

Reply:
cool!